sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -498,6 +498,22 @@ class SessionCatalog(
   }
 
   /**
+   * Invalidate all the cache entries.
+   */
+  def invalidateCache(): Unit = { /* no-op */ }
```

Comment:
before we move the cache to `sql/core`, these 3 APIs are only useful under hive context right?